Homework5/Routers/Routers/Graph.cs

```diff
@@ -0,0 +1,205 @@
+﻿namespace Routers;
+
+/// <summary>
+/// Class for graph and methods for work with it
+/// </summary>
+public class Graph
+{
+    private int[,] matrix;
+    private int sizeOfMatrix = 5;
+    private List<int> nodes;
+    private Dictionary<(int, int), int> edges;
+
+    public Graph(string pathInputFile)
+    {
+        nodes = new();
+        edges = new();
+        matrix = new int[sizeOfMatrix, sizeOfMatrix];
+        Initialization(pathInputFile);
+    }
+
+    private void Initialization(string pathInputFile)
+    {
+
+        using StreamReader inputFile = new(pathInputFile);
```

Comment:
```suggestion
    {
        using StreamReader inputFile = new(pathInputFile);
```
После открывающей фигурной скобки пустая строка не ставится